sdk/client-core/core/src/main/java/io/clientcore/core/util/ClientLogger.java

```diff
@@ -736,38 +599,4 @@ public String toString() {
             return caseSensitive;
         }
     }
-
-    /**
-     * Removes CR, LF or CRLF pattern in the {@code logMessage}.
-     *
-     * @param logMessage The log message to sanitize.
-     * @return The updated logMessage.
-     */
-    private static String removeNewLinesFromLogMessage(String logMessage) {
```

Comment:
Why did we remove this?

Reply:
it's now part of json serialization `jsonWriter.writeStartObject().writeStringField("message", message);` on line 468